infrastructure/images/moduletest-reset-jobs/moduletest-files-reset-job.mjs

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env zx
+
+const projectName = $1;
+if (!projectName) {
+  throw Error("No 'projectName' provided");
+}
+
+console.log(`Reseting files from ${projectName}-main to ${projectName}-moduletest
+  \n
+  Will now delete all files and folder in '/app/web/sites/default/files'
+  \n
+`);
+
+try {
+  // This will throw as there's a long running PHP process that keeps using
+  // a file in a /php folder inside the CLI container
+  // Emptying the folder is however successfull and we should ensure the
+  // program doesn't exit;
+  await $`kubectl exec -n ${projectName}-moduletest deploy/cli -- bash -c rm -fr /app/web/sites/default/files`
+} catch(error) {
+  if(error.exitCode != 1) {
+    throw Error("unexpected error", error.stderr);
+  }
+  console.log("As expected, the deletion of all files and folders in '/app/web/default/files' threw an 'exit 1'", error);
+}
+
+console.log(`Will now move files from ${projectName}-main to ${projectName}-moduletest`);
+
+try {
+  await $` kubectl exec -n ${projectName}-main deploy/cli -- tar cf - /app/web/sites/default/files | kubectl exec -i -n ${projectName}-moduletest deploy/cli -- tar xvf - -C /`;
+} catch(error) {
+  console.log("file move failed", error.stderr);
```

Comment:
You should probably throw the error here, so you don't emit the "file reset complete"-message.

Reply:
Oh.. sorry, that was also my intention - fixing

Reply:
Consider the `Error` constructor - you can't just pass in everything. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error

Reply:
done: https://github.com/danskernesdigitalebibliotek/dpl-platform/pull/634/commits/de6f239b0a98d16ecde448635c7286e709b3e59d

Reply:
You're not passing the error directly into cause - you're taking stdout from the errors. Is this intentional?

Reply:
yes, because that is where the actual error message resides: https://google.github.io/zx/getting-started#usage

Reply:
But that's not how the `cause`-option works. You pass the original error - not a string message.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause

Reply:
https://github.com/danskernesdigitalebibliotek/dpl-platform/pull/634/commits/de6f239b0a98d16ecde448635c7286e709b3e59d